src/govuk/components/character-count/character-count.js

```diff
@@ -20,15 +20,27 @@ CharacterCount.prototype.init = function () {
   // Check for module
   var $module = this.$module
   var $textarea = this.$textarea
-  var $countMessage = this.$countMessage
+  var $fallbackLimitMessage = this.$fallbackLimitMessage
```

Comment:
Is this declaration necessary? You could replace the instances of `$fallbackLimitMessage` below with `this.$fallbackLimitMessage`. It doesn't look like this is being used anywhere else in the page so the risk of mutating the var right in the `this` is pretty low as far as I can tell.

You could go one step further and remove the `if` from the module declaration on line 9 and only scope the variable to the `init` function.